### PR TITLE
Lesson 11. String and IO

### PR DESCRIPTION
**Pull request by feeL67**

### CODE REVIEW

File: src/com/tms/lesson11/SerializationService.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        ObjectOutputStream oos = null;
+        try {
+            oos = new ObjectOutputStream(new FileOutputStream("C:\\Users\\Lenovo\\Desktop\\UserSave.sav"));
+            oos.writeObject(user);
+        } catch (IOException exc) {
+            exc.printStackTrace();
+        } finally {
+            if (oos != null) {
+                try {
+                    oos.close();
```
Comment (gr00shik) [inline src/com/tms/lesson11/SerializationService.java:20]: you can use try-with-resourses as for previous class

File: src/com/tms/lesson11/SplitServ.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        StringBuilder romeo = new StringBuilder("Romeo words");
+        StringBuilder julietta = new StringBuilder("Julietta words");
+
+        try (BufferedReader br = new BufferedReader(new FileReader("C:\\Users\\Lenovo\\Desktop\\Romeo&Juliette.txt"))) {
+            String s = br.readLine();
+            while (s != null) {
+                s = br.readLine();
+                if (s == null) {
+                    br.close();
+                } else if (s.startsWith("Romeo")) {
```
Comment (gr00shik) [inline src/com/tms/lesson11/SplitServ.java:17]: what about next example?
Romeo: my first text on line1
              my next text on line2

**Review by gr00shik — COMMENTED** [review]
ok)

File: src/com/tms/lesson11/SerializationService.java (hunk 1 above)
Comment (feeL67) [reply src/com/tms/lesson11/SerializationService.java:20]: fixed

File: src/com/tms/lesson11/SplitServ.java (hunk 2 above)
Comment (feeL67) [reply src/com/tms/lesson11/SplitServ.java:17]: fixed)

File: src/com/tms/lesson11/SplitServ.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+public class SplitServ {
+    public static void main(String[] args) {
+
+        StringBuilder romeo = new StringBuilder("Romeo words");
+        StringBuilder julietta = new StringBuilder("Julietta words");
+        StringBuilder common = null;
+        String line;
+
+        try (BufferedReader br = new BufferedReader(new FileReader("C:\\Users\\Lenovo\\Desktop\\Romeo&Juliette.txt"))) {
+            while ((line = br.readLine()) != null) {
```
Comment (gr00shik) [inline src/com/tms/lesson11/SplitServ.java:14]: it is not a good approach, memory may be empty)

**Review by gr00shik — COMMENTED** [review]
ok

File: src/com/tms/lesson11/SplitServ.java (hunk 3 above)
Comment (feeL67) [reply src/com/tms/lesson11/SplitServ.java]: Done

**Pull request merged**